scripts/pack_server.sh

```diff
@@ -121,14 +114,26 @@ copy_file ./rdsn/thirdparty/output/lib/libk5crypto.so.3 ${pack}/bin
 copy_file ./rdsn/thirdparty/output/lib/sasl2 ${pack}/bin
 copy_file ./scripts/sendmail.sh ${pack}/bin
 copy_file ./src/server/config.ini ${pack}/bin
+copy_file ./src/server/config.min.ini ${pack}/bin
+copy_file ./config_hdfs.sh ${pack}/bin
 
-copy_file `get_stdcpp_lib $custom_gcc` ${pack}/bin
-copy_file `get_system_lib server snappy` ${pack}/bin/`get_system_libname server snappy`
-copy_file `get_system_lib server crypto` ${pack}/bin/`get_system_libname server crypto`
-copy_file `get_system_lib server ssl` ${pack}/bin/`get_system_libname server ssl`
-copy_file `get_system_lib server aio` ${pack}/bin/`get_system_libname server aio`
-copy_file `get_system_lib server zstd` ${pack}/bin/`get_system_libname server zstd`
-copy_file `get_system_lib server lz4` ${pack}/bin/`get_system_libname server lz4`
+copy_file "$(get_stdcpp_lib $custom_gcc)" "${pack}/bin"
+
+pack_system_lib() {
+    SYS_LIB_PATH=$(get_system_lib server "$1")
+    if [ -z "${SYS_LIB_PATH}" ]; then
+        echo "ERROR: library $1 is missing on your system"
+        exit 1
+    fi
+    SYS_LIB_NAME=$(get_system_libname server "$1")
+    copy_file "${SYS_LIB_PATH}" "${pack}/bin/${SYS_LIB_NAME}"
```

Comment:
why not `copy_file "${SYS_LIB_PATH}" "${pack}/bin"`?

Reply:
I'm not 100% sure but `${SYS_LIB_NAME}` and `${SYS_LIB_PATH}` could point to different file names.